### DISCUSSION TITLE
Feature Request: Proxy or IPv4 Load Balancer Support

### DISCUSSION THREAD

**Discussion by dazzypark** [TopiSenpai/LavaSrc#246] [General]
First, I’d like to thank you for the incredible work you’ve done with the LavaLink and LavaSrc plugins — they’ve been fantastic!

I’m currently working with the Deezer platform for music streaming, and I’ve encountered an issue where Deezer temporarily blocks my IP due to high usage. To prevent these temporary blocks and to ensure smooth streaming, I’m wondering if it would be possible to integrate a proxy or IPv4 load balancing feature within LavaSrc.

I have around 4 unused IPv4 addresses and would like to use them efficiently to distribute the load and avoid hitting rate limits from Deezer.

Would it be possible to add a feature to handle IP rotation or load balancing across multiple IPv4 addresses to avoid these blocks? This would help ensure continuous, uninterrupted service.

Thank you again for your hard work, and I look forward to hearing from you!

**Comment by topi314** [comment]
the easiest way to get around this is to run multiple lavalink servers on different ips.
I don't mind having http proxy support but doing loadbalancer stuff like youtube could work too

**Reply by dazzypark** [reply]
Oh, that sounds great! I’d love to run multiple Lavalink servers on different IPs, but I’ve already purchased a 16-core VPS for a whole year 😅. Do the ratelimit settings in the Lavalink configuration work with LavaSrc as well?

**Reply by topi314** [reply]
> Do the ratelimit settings in the Lavalink configuration work with LavaSrc as well?

no
